docs/source/dev/login_systems.rst

```diff
@@ -0,0 +1,119 @@
+Login systems
+=============
+
+Bonsai currently support two login systems, LDAP authentication and a simple username and password authentication. These login sytems are mutually exclusive so by chosing one the other is being disabled.
+
+Simple authentication
+---------------------
+
+The default authentication method is logging using a username and password. The password is being salted and stored in the mongo database.
+
+LDAP authentication
+-------------------
+
+Authentication can be made against an institutional LDAP3 server. You need an existing LDAP authentication server to user this authentication method. The users need to have an account in Bonsai in addition to having an entry on the LDAP server, because the user roles are determined from the Bonsai account.
+
+Use either the API CLI or the Admin panel to create a new user.
+
+The LDAP3 server connection is configured using environmental variables. At minimum you need to configure the following variables.
+
+.. csv-table:: Minimal configuration :rst:dir:`csv-table`
+   :header: "Variable name", "Description"
+
+   "LDAP_HOST",               "Server host or IP"
+   "LDAP_PORT",               "Server port"
+   "LDAP_BASE_DN",            "Base DN for searching the server"
```

Comment:
Maybe write out Distinguished Name (DN) first time mentioned (I needed to google)